From 461db48fc56a8521a78314e5f02ed516ed76b1ee Mon Sep 17 00:00:00 2001
From: innovara <fombuena@outlook.com>
Date: Fri, 27 Oct 2023 11:53:49 +0100
Subject: [PATCH 08/10] fix(pkcs11): delete trailing dot on
 libcryptsetup-token-systemd-pkcs11.so

libcryptsetup-token-systemd-pkcs11.so.* doesn't match the file libcryptsetup-token-systemd-pkcs11.so

(Cherry-picked commit 5edaca7637d76a05429a31c28304ef3f476fe2f9 from PR#2540)
---
 modules.d/91pkcs11/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/91pkcs11/module-setup.sh b/modules.d/91pkcs11/module-setup.sh
index 547631db..5675efb5 100755
--- a/modules.d/91pkcs11/module-setup.sh
+++ b/modules.d/91pkcs11/module-setup.sh
@@ -30,6 +30,6 @@ install() {
         {"tls/$_arch/",tls/,"$_arch/",}"libffi.so.*" \
         {"tls/$_arch/",tls/,"$_arch/",}"libp11-kit.so.*" \
         {"tls/$_arch/",tls/,"$_arch/",}"libcryptsetup.so.*" \
-        {"tls/$_arch/",tls/,"$_arch/",}"/cryptsetup/libcryptsetup-token-systemd-pkcs11.so.*"
+        {"tls/$_arch/",tls/,"$_arch/",}"/cryptsetup/libcryptsetup-token-systemd-pkcs11.so*"
 
 }
-- 
2.42.0

